src/changelog.js

```diff
@@ -414,8 +414,7 @@ async function getFormattedPullRequestsForDependencies({owner, repo, fromTag, to
     }
   }
 
-  const results = await Promise.all(resultPromises)
-  return results.join('\n\n')
+  return results.join('\n')
```

Comment:
Does this result in poor spacing between the package sections or does it seem OK?

Reply:
Wow that's weird, I didn't mean to do that! Good catch.